numba/core/lowering.py

```diff
@@ -983,7 +983,15 @@ def _lower_call_normal(self, fnty, expr, signature):
             argvals = self.fold_call_args(
                 fnty, signature, expr.args, expr.vararg, expr.kws,
             )
-        impl = self.context.get_function(fnty, signature)
+        tname = expr.hardware
+        if tname is not None:
+            from numba.core.extending_hardware import (dispatcher_registry,
+                                                       hardware_registry)
+            disp = dispatcher_registry[hardware_registry[tname]]
```

Comment:
The pattern of `dispatcher_registry[hardware_registry[tname]]` has appeared multiple times. It's common enough to have a method for it.

Reply:
Yes! Done.